### PR TITLE
Add new MOA Chat to Crosshatch

### PR DESCRIPTION
**Pull request by mr-spaghetti-code**
The goal is to make this available on Decent as a selectable mix.

It's the same MOA as the example provided in moa-llm 

TESTED=LOCALLY

### CODE REVIEW

**Comment by mr-spaghetti-code** [comment]
This should only be merged when https://github.com/catena-labs/monorepo/pull/1299 is in.

File: mix-definitions/moa-chat/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Chat Mixture of Agents
+
+This is a Mixture-of-Agents (MoA) synthesis mix optimized for chat. This mix leverages two 'proposer' models (Llama 3.1 70B on Groq and DeepSeek v2.5) with an 'aggregation' model (Llama 3.1 70B) that synthesizes their outputs. The goal is to produce fast and balanced responses by sourcing responses from multiple models.
+
+This model is availale on [Decent](https://www.decent.xyz), an open-source chat client on Android and iOS.
```
Comment (domleboss97) [inline mix-definitions/moa-chat/README.md:5]: nit: small typo here.

File: mix-definitions/moa-chat/index.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    ]
+  },
+  cost: {
+    inputCostPerUnit: 0.00000046,
+    outputCostPerUnit: 0.00000064,
+    unit: "token"
+  },
+  createdAt: new Date("2024-10-21T13:00:00-04:00"), // August 26, 2024 1pm Eastern
+  description:
+    "A Mixture-of-Agents (MoA) synthesis mix optimized for fast and balanced chat usecases. This mix leverages multiple 'proposer' models, including Llama 3.1 70B on Groq and DeepSeek v2.5, with an 'aggregation' model that synthesizes their outputs.",
```
Comment (domleboss97) [inline mix-definitions/moa-chat/index.ts:33]: nit: `use case` should be 2 words

**Review by domleboss97 — COMMENTED** [review]
Should we name this one `moa-chat-os` since it's only open source models? There may be a better chat combination with proprietary models right?

**Comment by mr-spaghetti-code** [comment]
Yep, you're right. I'll do that before merging

**Review by domleboss97 — APPROVED** [review]

**Pull request merged**